packages/react/src/components/List/HierarchyList/HierarchyList.jsx

```diff
@@ -138,9 +139,7 @@ const defaultProps = {
   getAllowedDropIds: null,
   onListUpdated: () => {},
   cancelMoveClicked: () => {},
-  itemWillMove: () => {
-    return true;
-  },
```

Comment:
I think the snapshots are still there, it is just a lot less code now since we are not rendering the HierarchyListReorderModal unless there is a selection made.